src/pods/test-multiple-choice/test-multiple-choice.business.ts

```diff
@@ -0,0 +1,36 @@
+import { VerbEntityGlobal } from 'core/verbs';
+import { VerbQuiz } from './test-multiple-choice.vm';
+import { Verb, VerbCorrect, createDefaultVerbCorrect } from 'common/model';
+
+// TODO: maybe add some defensive programming here? edge cases / errors ?
```

Comment:
This two TODO we should decide whether to remove them or do something about them

Reply:
I've removed both TODO's and moved **pickRandomVerb** function to common